src/frontend/screens/Observation/index.tsx

```diff
@@ -61,7 +66,13 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
   const {lat, lon, createdBy} = observation;
   const isMine = deviceId === createdBy;
   // Currently only show photo attachments
-  const photos = [];
+  const attachmentUrls = useAttachmentUrlQueries(
+    observation.attachments,
+    'original',
```

Comment:
Shouldn't we be requesting the `'thumbnail'`?

Reply:
Good point, didn't know this is a thing to be honest. That's changed now.